### PR TITLE
Lesson 7 alexandra siriciuc

### PR DESCRIPTION
**Pull request by AlexandraSRC**
Homework decorator function

### CODE REVIEW

File: lesson_7/assignment.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+### Let's practice
```
Comment (mariuspodean) [inline lesson_7/assignment.md]: This is the requirements file from the previous assignment. Please replace it with the current one :)

File: lesson_7/alexandra_siriciuc/homework decorators.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+def uppercase_decorator(function):
+        def inner_function(arg):
+            func = function(arg)
```
Comment (mariuspodean) [inline lesson_7/alexandra_siriciuc/homework decorators.py:3]: This works as expected, except that the variable names are a bit deceiving.
Here you have a variable called func, when this is actually the string returned by the call of the original function. Please rename to something that better reflects the data it holds.

**Review by mariuspodean — COMMENTED** [review]
Good job!

File: lesson_7/alexandra_siriciuc/homework decorators.py (hunk 2 above)
Comment (Liitle) [reply lesson_7/alexandra_siriciuc/homework decorators.py:3]: @AlexandraSRC am marcat comentariul lui Marius de mai sus cu fisierul de assignment resolved. Dupa ce modifici numele variabilei putem sa facem merge la tema.

**Review by Liitle — APPROVED** [review]
Good job! 🥳

**Pull request merged**